engine/testabilities/testmode/db_mode.go

```diff
@@ -0,0 +1,45 @@
+/*
+Package testmode provides functions to set special modes for tests,
+allowing to use actual Postgres or SQLite file for testing, especially for development purposes.
+Important: It should be used only in LOCAL tests.
+Calls of SetPostgresMode and SetFileSQLiteMode should not be committed.
+*/
+package testmode
+
+import (
+	"os"
+	"testing"
+)
+
+const (
+	modeEnvVar = "TEST_DB_MODE"
+	nameEnvVar = "TEST_DB_NAME"
+)
+
+// SetPostgresMode sets the test mode to use actual Postgres and sets the database name.
+func SetPostgresMode(t testing.TB, dbName string) {
+	t.Setenv(modeEnvVar, "postgres")
+	t.Setenv(nameEnvVar, dbName)
+}
+
+// SetFileSQLiteMode sets the test mode to use SQLite file
+func SetFileSQLiteMode(t testing.TB) {
+	t.Setenv(modeEnvVar, "file")
+}
+
+// CheckPostgresMode checks if the test mode is set to use actual Postgres and returns the database name.
+func CheckPostgresMode() (ok bool, dbName string) {
+	if os.Getenv(modeEnvVar) != "postgres" {
+		return false, ""
+	}
+	dbName = os.Getenv(nameEnvVar)
+	if dbName == "" {
```

Comment:
I don't like to be forced to set it, I want it to default to postgres as it was previously.

Reply:
I'm adding `SetPostgresMode` and `SetPostgresModeWithName`

Reply:
Done 👍